ticket_dispenser.c

```diff
@@ -1,149 +1,203 @@
-#include <Wire.h>
-
+#include "Wire.h"
 #include "Keypad.h"
 #include "Adafruit_LEDBackpack.h"
 #include "Adafruit_GFX.h"
 
 Adafruit_7segment matrix = Adafruit_7segment();
-
+//
 // Setup the keypad
-const byte ROWS = 4; //four rows
-const byte COLS = 3; //three columns
+//
+const byte ROWS = 4;
+const byte COLS = 3;
 char keys[ROWS][COLS] = {
-  {1,2,3},
-  {4,5,6},
-  {7,8,9},
-  {'*','^','#'}
+	{1,2,3},
+	{4,5,6},
+	{7,8,9},
+	{'*','^','#'}
 };
```

Comment:
This is a matrix - its a multi-dimensional array, organized into rows and columns. Just thought it was cool you were using one in this code, as we were just chatting about the talk I'm giving at Rubyconf